index.html

```diff
@@ -2163,14 +2163,14 @@
       <div class="modal-content">
         <div class="well">
           <h3>Strips metadata from video file</h3>
-          <p><code>ffmpeg -i <i>input_file</i> -map_metadata -1 -vcodec copy -acodec copy test-nometadata.mov</code></p>
+          <p><code>ffmpeg -i <i>input_file</i> -map_metadata -1 -c:v copy -c:a copy <i>input_file</i></code></p>
```

Comment:
Ah, turns out I wasn't sleepy but trying to work during a meeting... ;) One moment!